modules/build.gradle

```diff
@@ -29,7 +29,7 @@ ext {
 		"com.liferay.gradle.plugins.css.builder.version": "5.0.1",
 		"com.liferay.gradle.plugins.rest.builder.version": "1.1.21",
```

Comment:
Do we still need this token anymore? `"com.liferay.gradle.plugins.rest.builder.version": "1.1.21",`

Reply:
Removed